check_source.py

```diff
@@ -187,8 +193,60 @@ def check_source_submission(self, source_project, source_package, source_revisio
             elif self.repo_checker is not None:
                 self.add_review(self.request, by_user=self.repo_checker, msg='Please review build success')
 
+        if self.has_whitelist_warnings(source_project, source_package, target_project, target_package):
+            # if there are any add a review for the security team
+            # maybe add the found warnings to the message for the review
+            self.add_review(self.request, by_group='security-team', msg="The package is submitted to an official product and it has warnings that indicate that it need to go through a security review. Those warnings can only be ignored in devel projects. For more information please read: {}.".format(CheckSource.AUDIT_BUG_URL))
```

Comment:
`security-team` does not exist on IBS so this will cause crashes there and based on your use-case you likely want it disable on IBS. In general everything should be configurable via the configuration tooling already available. Add the security team review to `target_project_config()` following the convention there. Additionally, an option similar to `check-source-add-review-team` should be provided with the default set to false for Leap in `osclib/conf.py` (and we'll need to make sure both active SLE targets remote configs are update since they do not use defaults.

Reply:
No we want this in IBS and OBS. I was suprised to learn that we don't have the same group setup in IBS and will have this changed so we have the group in both instances